frame/support/test/tests/benchmark_ui/extrinsic_call_out_of_fn.stderr

```diff
@@ -0,0 +1,15 @@
+error: macros that expand to items must be delimited with braces or followed by a semicolon
```

Comment:
Huh, how does this error show up?

Reply:
This is if you use the macro somewhere outside of where it is supposed to be used -- it expands to a compile error in that scenario. Looking into if I can get it to go away

Reply:
was a missing semi in the quoted part 😆